src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days < 3) {
+    return totalCost;
+  } else if (days >= 3 && days <= 6) {
+    return (totalCost = totalCost - 20);
```

Comment:
There's no need to reassign `totalCost` here; simply return `totalCost - 20`. Reassignment in a return statement is unnecessary and can be confusing.